study.md

```diff
@@ -43,21 +42,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+Developers are referring to the model-view-controller when referring to the word "view".  Though I'm still not entirely sure what the MVC is, it does work on a single application page in order to interact with a server to product api - content.  
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+I think the main advantage is simplicity of design which allows developers to really make a lean, web-based data processing machine.  Instead of having to worry about design and linking between multi-pages, a SPA puts all available information of a site on a single page.
```

Comment:
It also makes the user's experience smoother as the page doesn't have to completely reload all the time